bodhi/server/models.py

```diff
@@ -1506,6 +1506,21 @@ def get_testcase_karma(self, testcase):
                         bad += 1
         return bad * -1, good
 
+    def beautify_title(self, amp=False):
+        if len(self.builds) == 1:
```

Comment:
I am not an expert but IMHO first and second condition can be merged because `join()` method doesn't add separator when length of iterable argument equals to one and you also can use slicing with numbers higher than actual length of list. For example:
```
>>> arr1 = ['AAA']
>>> arr2 = ['AAA', 'BBB']
>>> arr3 = ['AAA', 'BBB', 'CCC']
>>> arr4 = ['AAA', 'BBB', 'CCC', 'DDD']
>>> ' and '.join(arr1[:2])
'AAA'
>>> ' and '.join(arr2[:2])
'AAA and BBB'
>>> ' and '.join(arr3[:2])
'AAA and BBB'
>>> ' and '.join(arr4[:2])
'AAA and BBB'
```